bezier-rs/docs/interactive-docs/src/utils/drawing.ts

```diff
@@ -8,8 +8,8 @@ export const getContextFromCanvas = (canvas: HTMLCanvasElement): CanvasRendering
 	return ctx;
 };
 
-export const drawLine = (ctx: CanvasRenderingContext2D, p1: Point, p2: Point): void => {
-	ctx.strokeStyle = "grey";
+export const drawLine = (ctx: CanvasRenderingContext2D, p1: Point, p2: Point, stroke = "grey"): void => {
```

Comment:
Please use the spelling "gray" for consistency with the spelling used in other parts of the project. There's another